Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Culling.h

```diff
@@ -290,21 +290,18 @@ namespace AZ
             //! Returns the number of cullables that have been added to the CullingScene
             uint32_t GetNumCullables() const;
 
-            CullingDebugContext& GetDebugContext()
-            {
-                return m_debugCtx;
-            }
+            CullingDebugContext& GetDebugContext();
 
             //! Returns the visibility scene
-            const AzFramework::IVisibilityScene* GetVisibilityScene() const { return m_visScene; }
+            const AzFramework::IVisibilityScene* GetVisibilityScene() const;
 
         protected:
             size_t CountObjectsInScene();
 
         private:
-            void BeginCullingTaskGraph(const AZStd::vector<ViewPtr>& views);
-            void BeginCullingJobs(const AZStd::vector<ViewPtr>& views);
-            void ProcessCullablesCommon(const Scene& scene, View& view, AZ::Frustum& frustum, void*& maskedOcclusionCulling);
+            void BeginCullingTaskGraph(const Scene& scene, const AZStd::vector<ViewPtr>& views);
+            void BeginCullingJobs(const Scene& scene, const AZStd::vector<ViewPtr>& views);
```

Comment:
repeat: These vector parameters can be made spans
```suggestion
            void BeginCullingTaskGraph(const Scene& scene, AZStd::span<const ViewPtr> views);
            void BeginCullingJobs(const Scene& scene, AZStd::span<const ViewPtr> views);
```